org.scala.tools.eclipse.search/src/org/scala/tools/eclipse/search/indexing/Index.scala

```diff
@@ -79,6 +81,15 @@ trait Index extends HasLogger {
     location(project).toFile.exists
   }
 
+  def indexIsClean(project: IProject): Boolean = {
```

Comment:
Uupsi, the name is good, it does exactly what the name implies. Please, disregard my comment.